### DISCUSSION TITLE
Does Jackson JR allow ALLOW_UNQUOTED_CONTROL_CHARS

### DISCUSSION THREAD

**Discussion by lmajano** [FasterXML/jackson-jr#168] [Q&A] [answered]
I can't find a way to enable this setting `ALLOW_UNQUOTED_CONTROL_CHARS`??

**Comment by cowtowncoder** [comment, marked as answer]
You will need to configure underlying `JsonFactory` in use: construct `JSON` with properly configured factory and that should just work.
